src/components/component-rule/component-rule.module.scss

```diff
@@ -1,7 +1,7 @@
 @import "../../styles/mixins/typography";
 
-.vibe-sb-comps-component-rule {
-  &_component {
+.componentRule {
+  &.component {
```

Comment:
I'm afraid this construction won't work here - same for most of the other folded classes

When you write 
```
.componentRule {
  &.component {
``` in SASS it's converting into `.componentRule.component` in CSS
Which targets elements which have classes `componentRule` AND `component` on the same element, which is not the case

You just need to remove the `&` sign and it'll be good :)